### PR TITLE
Add OneVM

### PR DESCRIPTION
**Pull request by BoryaGames**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1963,5 +1963,6 @@ HTML Themes and UI Kits to create your applications faster!
   * [SimpleRestore](https://simplerestore.io) - Hassle-free MySQL backup restoration. Restore MySQL backups to any remote database without code or a server.
   * [360Converter](https://www.360converter.com/) - Free tier useful website to convert: Video to Text && Audio to Text && Speech to Text && Real-time Audio to Text && YouTube Video to Text && add Video Subtitle. Maybe it will be helpful in a short video conversion or in a short youtube tutorial:)
   * [QRCodeBest](https://qrcode.best/) - Create custom QR codes with 13 templates, full privacy, and personal branding. Features tracking pixels, project categorization, and unlimited team seats on QRCode.Best.
+  * [OneVM](https://onevm.uk.to/) - KVM VPS, 1 GB RAM, 1 vCore, 15 GB storage, 10 ports + 1 SSH port, 350 mbps ethernet, free 20 hours/day.
```
Comment (ripienaar) [inline README.md:1966]: I can’t see any pricing info on this site to verify the stated offer

**Comment by RealPlurr** [comment]
Hello, PlurrYT here.
We actually state that we offer 1gb ram 1vcore 15gb storage and 20coins (20 hours) each 24h
image

File: README.md (hunk 1 above)
Comment (RealPlurr) [reply README.md:1966]: Please check reply: https://github.com/ripienaar/free-for-dev/pull/3429#issuecomment-2227387145

**Comment by ripienaar** [comment]
ok I read again all these coins etc it was confusing and I didn’t see any detailed pricing page for clarification

**Comment by ripienaar** [comment]
Thanks

**Comment by RealPlurr** [comment]
> ok I read again all these coins etc it was confusing and I didn’t see any detailed pricing page for clarification

Thanks for feedback, we will make sure to make it more clear for users!

**Comment by PlurrYT-Kolama** [comment]
This needs to get link updated to https://wiki.onevm.uk.to/en/start 
Landing page seems to be deleted

**Comment by FatGrizzly** [comment]
"seems to be deleted"

You're the one running this service aren't you?

whats the usage of a freedns afraid domain for a production service?

@ripienaar I think we should remove this from listing totally.

**Comment by ripienaar** [comment]
Yeah I think I will pass. It never really looked above board or sustainable long run.

**Pull request merged**